.github/workflows/ci.yml

```
@@ -11,19 +11,24 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby-version: ['3.0', '3.1', '3.2', '3.3']
-        gemfile: ['rails_6.0','rails_6.1','rails_7.0', 'rails_7.1']
+        ruby-version: ['3.0', '3.1', '3.2', '3.3', '3.4']
+        gemfile: ['rails_6.1', 'rails_7.0', 'rails_7.1', 'rails_8.0']
```

Does `rails_7.2` need to be mentioned here?

```suggestion
        gemfile: ['rails_6.1', 'rails_7.0', 'rails_7.1', 'rails_7.2', 'rails_8.0']
```

---

yup, missed this too